hack/build/operator/build

```diff
@@ -40,7 +40,6 @@ gitHash="github.com/coreos/etcd-operator/version.GitSHA=${GIT_SHA}"
 go_ldflags="-X ${ldKVPairs} -X ${gitHash}"
```

Comment:
Change this script to only do build. Now it also does push.

Reply:
@hongchaodeng do you want me to do the above in this pr or a separate pr?

Reply:
Do it in separate pr.